packages/select-react/src/Select.tsx

```diff
@@ -115,18 +115,36 @@ export const Select = forwardRef<HTMLSelectElement, SelectProps>(
         const [selectedValue, setSelectedValue] = useState(value);
         const hasSelectedValue = typeof selectedValue !== "undefined" && selectedValue !== "";
 
+        const isSearchable = !!searchable;
+
+        const searchFn = (item: ValuePair) => {
+            if (item.label.toLowerCase().includes(searchValue.toLowerCase())) {
+                return true;
+            }
+
+            if (typeof searchable === "function") {
```

Comment:
Skal funksjonen få presedens over det "innebyggede" søket, flyttes før `if (item.label.toLowerCase osv...`?

Reply:
Tanken bak å sette det sånn er at man ikke trenger å huske å legge inn label-sjekken selv. Jeg ser ikke helt for meg at det er ønskelig å override så da tenkte jeg at den like greit kunne komme i etterkant.

Innvendinger? :D